samples/cpp/speech_sample/main.cpp

```diff
@@ -83,26 +83,29 @@ int main(int argc, char* argv[]) {
         // -------------------------------------
         ov::runtime::Core core;
         slog::info << "Loading model files:" << slog::endl << FLAGS_m << slog::endl;
-        std::shared_ptr<ov::Model> model = core.read_model(FLAGS_m);
-        check_number_of_inputs(model->inputs().size(), numInputFiles);
-        const ov::Layout tensor_layout{"NC"};
-        ov::preprocess::PrePostProcessor proc(model);
-        for (int i = 0; i < model->inputs().size(); i++) {
-            proc.input(i).tensor().set_element_type(ov::element::f32).set_layout(tensor_layout);
-        }
-        for (int i = 0; i < model->outputs().size(); i++) {
-            proc.output(i).tensor().set_element_type(ov::element::f32);
+        uint32_t batchSize = (FLAGS_cw_r > 0 || FLAGS_cw_l > 0) ? 1 : (uint32_t)FLAGS_bs;
+        std::shared_ptr<ov::Model> model;
+        if (!FLAGS_m.empty()) {
```

Comment:
maybe it is better to add a few comments here, e.g. preprocessing steps are not applicable for the compiled model?

Reply:
ok

Reply:
Thanks, but this is a sample and we have to explain why we are skipping preprocessing steps for some cases, what about the following comment:
"The preprocessing steps can be done only for loaded network and are not applicable for the imported network (already compiled)?